src/app/dashboard-widgets/loading-widget/loading-widget.component.html

```diff
@@ -1,4 +1,6 @@
 <div class="f8-loading f8-blank-slate-card">
   <div class="spinner spinner-lg"></div>
-  <p>{{message}}</p>
+  <h3 *ngIf="title !== undefined">{{title}}</h3>
```

Comment:
These checks are curious. Does it make sense to also check for null?

Reply:
Just omitting the HTML if the title has not been provided.